sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -11,6 +13,33 @@
 LCClient = noaa.NOAAIndicesClient()
 
 
+def create_mock_unified_object(start_date, end_date):
+    '''
+    Let us create a mock QueryResponse object
+    using that, we will construct UnifiedResponse
+    We will prefill some downloaded data from
+    running noaa.NOAAIndicesClient().fetch(Time('2012/8/9', '2012/8/10'),
+                                                    Instrument('noaa-indices'))
+    '''
+    # Create a mock QueryResponse object
+    map_ = {}
+    map_['Time_start'] = parse_time(start_date)
+    map_['Time_end'] = parse_time(end_date)
+    map_['source'] = 'sdic'
+    map_['instrument'] = 'noaa-indices'
+    map_['physobs'] = 'sunspot number'
+    map_['provider'] = 'swpc'
+
+    resp = QueryResponse.create(map_, [''])
+    # Attach the client with the QueryResponse
+    resp.client = 'noaa-indices'
+
+    # Create a UnifiedResponse object
+    uresp = UnifiedResponse(resp)
+    print(type(uresp))
```

Comment:
There should no be `print`s on the tests.
```suggestion

```

Reply:
Removed the print statement.